src/wallet/wallet_args.cpp

```diff
@@ -138,7 +139,7 @@ namespace wallet_args
       log_path = command_line::get_arg(vm, arg_log_file);
     else
       log_path = mlog_get_default_log_path("monero-wallet-cli.log");
```

Comment:
wallet_rpc_server was re-initializing the logging code after this, and with the wrong path (current path, rather than where the binary was). wallet-rpc logs to console because of the new argument I pass (log_to_console), which is true only for monero-wallet-rpc.